sycl/include/sycl/ext/intel/experimental/esimd/detail/math_intrin.hpp

```diff
@@ -513,54 +492,55 @@ __esimd_dpas_inner(const __ESIMD_DNS::vector_type_t<RT, SZ> *src0,
   constexpr size_t SIMDSize = 8;
 #endif
 
-  constexpr bool
-      pvcHfDest = isPvc && std::is_same<RT, __ESIMD_EMU_DNS::half>::value,
-      pvcBfDest = isPvc && std::is_same<RT, short>::value,
-      pvcBfOrHfDest = pvcBfDest || pvcHfDest,
+  constexpr bool pvcHfDest =
+                     isPvc && std::is_same<RT, __ESIMD_EMU_DNS::half>::value,
+                 pvcBfDest = isPvc && std::is_same<RT, short>::value,
+                 pvcBfOrHfDest = pvcBfDest || pvcHfDest;
 
-      pvcBfDestChecks = pvcBfDest &&
-                        src1_precision == __ESIMD_ENS::argument_type::BF16 &&
-                        src2_precision == __ESIMD_ENS::argument_type::BF16,
+  bool pvcBfDestChecks = pvcBfDest &&
```

Comment:
why is this no longer constexpr?

Reply:
As `src1_precision` and `src2_precision` are changed to runtime argument (from template argument).